addons/xpaas/v3.10/xpaas-streams/datagrid72-image-stream.json

```diff
@@ -31,9 +32,30 @@
                             "version": "1.0",
                             "openshift.io/display-name": "Red Hat JBoss Data Grid 7.2"
                         },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-datagrid-7/datagrid72-openshift:1.0"
```

Comment:
To make this image stream to work, you should've login to the docker registry. On downstream we are doing it but upstream side we need to have some kind of start flag which will ask user to login to that registry before oc cluster up.

Reply:
@praveenkumar I see, thanks for pointing out. Do we have an issue for this? It seems relatively high priority to me.